test/hotspot/jtreg/compiler/vectorapi/TestVectorSlice.java

```diff
@@ -324,7 +324,7 @@ static void testDoubles() {
     }
 
     @Test
-    @IR(counts = {IRNode.VECTOR_SLICE, "9"})
+    @IR(counts = {IRNode.VECTOR_SLICE, "7"}, applyIfCPUFeature = {"sse2", "true"})
```

Comment:
How about separating the special cases (i.e. origin is `0/VLENGTH`), and use the `FailOn` check instead on them? Tests are more accurate.